Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.MatchRegex.cs

```diff
@@ -252,7 +252,7 @@ public void When_a_string_is_matched_and_the_expected_count_is_zero_and_string_m
 
                 // Assert
                 act.Should().Throw<XunitException>()
-                    .WithMessage($"Expected subject to match regex*\"a\" exactly 0 times, but found it 1 time.");
+                    .WithMessage($"Expected subject to match regex*\"a\" exactly 0 times, but found it 1 time in *{subject}*");
```

Comment:
Wouldn't it be shorter to just write `\"a\"` instead of `*{subject}*`.

Reply:
Yes..